renderer.js

```diff
@@ -86,6 +86,13 @@ num = st.slider("Select number", 1, 10, 3)
 
 st.write(f"{num}^{num}: ", num**num)
 `
+    } 
+    else if (message == "Snowpark") {
+        contents = fs.readFileSync('templates/snowpark.txt', 'utf8', (err, data) => {
```

Comment:
Excellent idea! Much better than hard-coding all the files in here! 

One suggestion -- what would you think of just making it a .py file? No harm, and it would give you free syntax highlighting when editing and viewing in github.

Reply:
Yes, I can probably do this! I changed it to .txt when I was trying to debug.

Reply:
Done.